types/clerk.io/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/export-just-namespace": "off"
```

Comment:
Please remove this file. See 
https://github.com/microsoft/dtslint/blob/master/docs/export-just-namespace.md for how. Thanks!